Ignoring overlapping pop-ups if present (no need to edit shortcut anymore) (#1)
Benehiko/nolvus-automagic-downloader: README.md

2. Choose the appropriate installer for your operating system (Windows, macOS, or Linux).
3. Run the downloaded installer and follow the on-screen instructions.
4. Confirm the installation by opening a terminal or command prompt and typing:
    ```powershell
    go version
    ```

5. You should see an output indicating the installed version of GoLang, confirming that the installation was successful.

### Nolvus Automagic Downloader Installation & Usage

1. **Build Nolvus Automagic Downloader**

    - Open **Windows Powershell** console

        - Press `Win + X` and select Windows PowerShell (or Windows Terminal if you have it installed).

    - Clone the Repository:

        ```powershell
        git clone git@github.com:Benehiko/nolvus-automagic-downloader.git
        cd nolvus-automagic-downloader
        ```

        - **Note:** If you aren't a skilled GitHub user but still want to download this wonderful utility, it might be easier for you to download the repository as a ZIP file (click the `< > Code` button at the top of the page and the `Download ZIP` button).

    - Build the Nolvus Automagic Downloader app:

        ```powershell
        go build .
        ```

>         - **Note:** If you encounter a build error saying that there is a problem with your Go version, open the `go.mod` file in your text editor and change the mentioned Go version so it matches your Go version.

Reviewer: ```suggestion
        - **Note:** If you encounter a build error saying that there is a problem with your Go version, open the `go.mod` file in your text editor and verify your version of Go is the same or greater.
```

Reviewer: wouldn't make sense to change the go version in `go.mod`. Go is usually backwards compatible, so upgrading to a later version of Go makes more sense.
